app/views/users/sessions.py

```diff
@@ -141,6 +144,95 @@ def add_speaker_view(event_id, session_id):
         return redirect(url_for('.index_view', event_id=event_id))
 
 
+@event_sessions.route('/download_speakers_as_csv/', methods=('GET', 'POST'))
+@belongs_to_event
+@can_access
+def download_speakers_as_csv(event_id):
+    sessions = DataGetter.get_sessions_by_event_id(event_id)
+    event = DataGetter.get_event(event_id)
+    data = "Speaker Name, Speaker Session(s) , Speaker Email, Speaker Status," \
+           " Speaker Mobile, Speaker Organisation, Speaker Position \n"
+
+    for session in sessions:
+        speakers = session.speakers
+        for speaker in speakers:
+            try:
+                data += speaker.name+','
+            except:
+                data += ','
+            try:
+                data += session.title+','
+            except:
+                data += ','
+            try:
+                data += speaker.email+','
+            except:
+                data += ','
+            try:
+                for sess in speaker.sessions:
+                    data += sess.state+','
+            except:
+                data += ','
+            try:
+                data += speaker.mobile+','
+            except:
+                data += ','
+            try:
+                data += speaker.organisation+','
+            except:
+                data += ','
+            try:
+                data += speaker.position+','
+            except:
+                data += ','
+            data += '\n'
+
+    response = make_response(data)
+    response.headers['Content-Type'] = 'text/csv'
+    response.headers['Content-Disposition'] = \
+        'inline; filename=%s-Speakers.csv' % (re.sub(r"[^\w\s]", '', event.name).replace(" ", "_"))
+
+    return response
+
+
+@event_sessions.route('/download_sessions_as_csv/', methods=('GET', 'POST'))
```

Comment:
I am not sure why POST method is allowed in this route. If POST request isn't required, it shouldn't be allowed.